src/assets/shared-styles/shared.module.scss

```diff
@@ -11,4 +11,64 @@
 .pageWrapper {
     text-align: center;
     margin-top: 1.45rem;
-}
\ No newline at end of file
+}
+
+.riskBuying {
+    width: 307px;
+    height: 115px;
+    font-size: 30px;
+    line-height: normal;
+    font-style: Cera-Light;
+    text-align: center;
+}
+
+.offenderRegistry {
+    width: 290px;
+    height: 60px;
+    font-size: 20px;
+    font-family: Cera-Bold;
+    line-height: normal;
```

Comment:
The text areas should not be given heights and widths--are you copying the direct styles from the mockups? It looks like it since there's so much repeating values here and things that don't quite make sense.

For things like text, try using padding and margin to give them the right spacing around other items. You don't need to set the font and font size for each class.  Aside from the height and width (which should not be used for text), all these classes are the same. Try using one class that styles all the text since the text looks the same. (except for the header). So each text part should look something like this:
`<p className= styles.consequence>Text here</p>`
and then your scss file would have something like
```
.consequence {
   font-family: cera-bold;
   padding: 1em;
   margin: 1em;
... (whatever here you need to make it look like the mockups)
}
```

Reply:
Changed styling